modules/UI/util/UIUtil.js

```diff
@@ -103,13 +127,13 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
             'top-right': 'sw'
         };
 
-        element.setAttribute("data-i18n", "[content]" + key);
-        APP.translation.translateElement($(element));
+        $(element).each(function () {
+            var el = $(this);
```

Comment:
@m-voloshin, I'm afraid I don't get the purpose of jQuery here. Could you please explain? As far as I understand the call sites of `setTooltip`, the value of `element` is of type `HTMLElement`. Then you do `$(element)` to get a jQuery selector that selects the one and only `element` and do an `each` loop+callback on a set of size equal to 1. Inside the callback, the original `element` value is now renamed for you to `this` so you wrap it a second time in a jQuery selector using `$(this)`. Why is all of this necessary? I see at least one wrapping is necessary because of `APP.translation.translateElement` (it beats me why the method name talks about element given that it expects a jQuery selector!)